Spatial.gd

```diff
@@ -284,27 +285,34 @@ func reset_minimap_masks():
 		minimap_path.material.set_shader_param("minimap_corner", compass_corner1)
 		minimap_path.material.set_shader_param("minimap_corner2", compass_corner2)
 
-var markerdata = Waypoint.Waypoint.new()
-var marker_file_dir = "user://protobins/"
+var Waypoint_data = Waypoint.Waypoint.new()
+var marker_file_dir = Directory.new()
 var marker_file_path = ""
-var marker_packs_array = Array()
+var root: TreeItem
+
+##########Node Connections###########
 onready var marker_packs = $Control/Dialogs/MarkerPacks/MarkerPacks
+onready var icons = $Icons
+onready var paths = $Paths
+onready var minimap = $Control/MiniMap
```

Comment:
lets add types to these variables because they are basically constant references to the godot scene nodes.
Is `onready const` a thing? I feel like it is not but it might be.